src/NeptuneSchema.js

```diff
@@ -271,26 +255,11 @@ async function getNodesProperties() {
 
 
 async function checkEdgeDirectionCardinality(d) {
-    let parameters = `{"from": "${d.from}", "edge": "${d.edge.label}", "to": "${d.to}"}`;
-    let queryFrom = `MATCH(from)-[r]->(to)
-                    WHERE $from in labels(from)
-                        AND type(r) = $edge
-                        AND $to in labels(to)
-                    WITH to, count(from) as rels
-                    WHERE rels > 1
-                    RETURN rels LIMIT 1`;
-    loggerLog(`Checking edge direction cardinality: ${queryFrom}`);
-    let responseFrom = await queryNeptune(queryFrom, parameters);
+    let queryFrom = `MATCH (from:${sanitize(d.from)})-[r:${sanitize(d.edge.label)}]->(to:${sanitize(d.to)}) WITH to, count(from) as rels WHERE rels > 1 RETURN rels LIMIT 1`;
+    let responseFrom = await queryNeptune(queryFrom);
     let resultFrom = responseFrom.results[0];
-    let queryTo = `MATCH(from)-[r]->(to)
-                    WHERE $from in labels(from)
-                        AND type(r) = $edge
-                        AND $to in labels(to)
-                    WITH from, count(to) as rels
-                    WHERE rels > 1
-                    RETURN rels LIMIT 1`;
-    loggerLog(`Checking edge direction cardinality: ${queryTo}`);
```

Comment:
Same as above

Reply:
Not intentional, added back a generic log message.